Apps/frontend/src/app.scss

```diff
@@ -78,8 +78,18 @@ nav {
   .icon-button {
     filter: invert(0) !important;
   }
+
+  table tr:nth-child(even) {
```

Comment:
Really like the idea of zebra colouring, but I would probably recomment alternating between light grey and white for light mode and [background-color == #242424] and a slightly brighter grey tone (maybe sth. like #404040.

Reply:
![image](https://user-images.githubusercontent.com/81998094/213917896-b4a08382-b72a-4add-806e-17694e2f0966.png)
 What about this?

Reply:
Definitely better. Can we use the respective main background colour (white and #242424) + light grey (for light mode) and dark grey (for dark mode), do you think that'd look good?

Reply:
Personally, I think it's actually okay that way.

Reply:
![image](https://user-images.githubusercontent.com/81998094/213926829-1feddb54-a6c9-41d5-b29e-d74aa5c8b63b.png)
 #242424 is to dark in my opinion so

#404040 is better I think
![image](https://user-images.githubusercontent.com/81998094/213926881-0ef35988-44f3-4006-a2b2-e5860fcc2afc.png)

Light mode is now:
![image](https://user-images.githubusercontent.com/81998094/213926911-358194cb-552c-4cd7-b421-d0952c3bf2a4.png)

Reply:
Your light mode is now fine with me, the lighter colour of the dark mode is a little too hefty imo, would prefer a little darker.

Fyi, my idea was for it to look sth. like this:
- Dark mode:
![image](https://user-images.githubusercontent.com/54808295/213929528-195542b5-4c7b-4543-bb0a-596992df1523.png)

-Light mode:
![image](https://user-images.githubusercontent.com/54808295/213929575-23e85724-9965-4991-8b84-399f4e3d4f6d.png)

Reply:
Would have my proposal ready, shall I commit?

Reply:
Yes just add it